src/confluent_kafka/schema_registry/_async/avro.py

```diff
@@ -57,12 +57,13 @@ async def _resolve_named_schema(
     named_schemas = {}
     if schema.references is not None:
         for ref in schema.references:
-            referenced_schema = await schema_registry_client.get_version(ref.subject, ref.version, True)
-            ref_named_schemas = await _resolve_named_schema(referenced_schema.schema, schema_registry_client)
+            # References in registered schemas are validated by server to be complete
+            referenced_schema = await schema_registry_client.get_version(ref.subject, ref.version, True)  # type: ignore[arg-type]
```

Comment:
Why is there a need for type ignoring here? Can we set the ref/referenced_schema type to avoid this?